README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
This code looks good just need to add null check for fail safe scenarios

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Brittle

Reply:
Fixed brittle fail safe scenarios - looks good

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Fault-tolerant

Reply:
Using Hibernate to execute a dynamic SQL statement built with user-controlled input can allow an attacker to modify the statement's meaning or to execute arbitrary SQL commands.